src/intersectionWith.js

```diff
@@ -12,7 +12,7 @@ var uniqWith = require('./uniqWith');
  * @memberOf R
  * @since v0.1.0
  * @category Relation
- * @sig (a -> a -> Boolean) -> [*] -> [*] -> [*]
+ * @sig ((a, a) -> Boolean) -> [a] -> [a] -> [a]
```

Comment:
nice find, @davidchambers; we almost rehashed that thread. shall we merge this one or wait for yours?